tests/unit/s2n_session_ticket_test.c

```diff
@@ -1098,6 +1119,7 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_connection_set_config(server_conn, server_config));
 
         /* Setup stuffers value containing the valid key name, valid iv and invalid encrypted blob */
+        POSIX_GUARD(s2n_stuffer_write_uint8(&server_conn->client_ticket_to_decrypt, S2N_PRE_ENCRYPTED_STATE_V1));
         POSIX_GUARD(s2n_stuffer_write_bytes(&server_conn->client_ticket_to_decrypt, ticket_key_name1, s2n_array_len(ticket_key_name1)));
 
```

Comment:
Don't you also need to write the info bytes? Is this test (and the other similar ones) passing for the wrong reason now? You have to be really careful with tests that test errors :/

Reply:
😔 Thanks for this catch. I'll double check all the decrypt_ticket tests that are supposed to fail.